hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/processes/InStreamHandler.java

```diff
@@ -0,0 +1,49 @@
+package org.molgenis.hadoop.pipeline.application.processes;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+/**
+ * Handles the streams retrieved from the process.
+ */
+public class InStreamHandler extends StreamHandler
+{
+	/**
+	 * Stores the write stream to be used.
+	 */
+	InputStream processStream;
+
+	/**
+	 * Object to which the output stream will be written to.
+	 */
+	InContainer inContainer;
+
+	/**
+	 * Initiates a new {@link InStreamHandler} instance that stores the required data.
+	 * 
+	 * @param outStream
+	 * @param inContainer
+	 */
+	InStreamHandler(InputStream processStream, InContainer inContainer)
+	{
+		this.processStream = processStream;
+		this.inContainer = inContainer;
+	}
+
```

Comment:
Always check that constructor args are not null with java.util.Objects.requireNonNull
See also GCC developer guidelines https://docs.google.com/document/d/1HrEp_u41GZ3rYD_iCIrnXleoyzD_eaPebVwa4cJe5L4/edit#